### ISSUE TITLE
node: pbkdf2-compat dropped compatibility for custom algorithm 3.0.0

### ISSUE THREAD

**Issue by dcousens** [crypto-browserify/pbkdf2-compat#7]
`3.0.0` was published without support for a custom hash algorithm in node.
As such, the defacto example (and main reason for this library before it was used in browserify) is now broken in Node 0.10.

``` javascript
var compat = require('pbkd2f-compat')
var derivedKey = compat.pbkdf2Sync('password', 'salt', 1, 32, 'sha512') // currently SHA1!!!
```

We need to merge #5 ASAP, and probably unpublish `3.0.0`.

**Comment by dcousens** [comment]
This is my fault for merging #4 and not making it clear that it was a WIP merge and not ready for publish.

**Comment by dcousens** [comment]
Merged #5, also applied style and indentation fixes including the use of https://github.com/feross/standard. 

Published as `3.0.1`.
